plans/tmt.fmf

```diff
@@ -2,10 +2,9 @@ summary:
     Run integration tests with tmt
 discover:
     how: fmf
-    url: https://github.com/psss/tmt
+    url: https://github.com/teemtee/tmt
     ref: fedora
     test: execute
-    filter: 'tier: 1, 2'
```

Comment:
@psss The test structure upstream has recently changed. Any clue on what tests are meant to be run here? `/tests/execture/reboot` seems like it should be included, but that one had `tier: 4`. Otherwise I don't see any `tier` other than `3` and `4`

Reply:
Just discussed this internally and came up with a new tag _beakerlib_, see [1].

While we touch the tmt plan definition in here, we could use the tag as well. However, we need to wait till it is merged.

1. https://github.com/teemtee/tmt/pull/2731